samples/pandafium.cc

```diff
@@ -223,16 +223,24 @@ static void WriteJpg(const char* pdf_name, int num, const void* buffer_void,
   /* Now we can initialize the JPEG compression object. */
   jpeg_create_compress(&cinfo);
   jpeg_stdio_dest(&cinfo, fp);
-  // Source data is B, G, R, unused.
+  // Source data is B, G, R, alpha.
   // Dest data is R, G, B.
   cinfo.image_width = image_width;      /* image width and height, in pixels */
   cinfo.image_height = image_height;
   cinfo.input_components = 4;           /* # of color components per pixel */
-  cinfo.in_color_space = JCS_EXT_BGRX;       /* colorspace of input image */
+  cinfo.in_color_space = JCS_EXT_BGRA;       /* colorspace of input image */
```

Comment:
I don't think this is strictly necessary because it seems like everything in https://github.com/libjpeg-turbo/libjpeg-turbo/blob/master/jccolor.c just ignores the alpha channel. I think it's more useful for the output mode. Still, seems good to document the format clearly, at least.